PatternPal/PatternPal.Tests/New_TestCasesRecognizers/Bridge/BridgeTestCase05.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage;
+using Console = System.Console;
+
+
+namespace PatternPal.Tests.New_TestCasesRecognizers.Bridge
+{
+
+    //This test is a possible bridge implementation.
+    /* Pattern:              Bridge
+     * Original code source:
+     *
+     * Requirements to fullfill the pattern:
+     *         Implementation interface or abstract class:
+     *            ✓  a) is an interface or abstract class <br/>
+     *            ✓  b) has at least one (abstract) method <br/>
+     *         Abstraction class:
+     *               a) has a private/protected field or property with the type of the Implementation interface or abstract class
+     *               b) has a method
+     *               c) has a method that calls a method in the Implementation interface or abstract class
+     *         Concrete Implementation
+     *            ✓  a) is an implementation of the Implementation interface or inherits from the 'Implementation' abstract class
+     *            ✓  b) if Implementation is an abstract class it should override it's abstract methods
+     *         Refined Abstraction:
+     *            ✓  a) inherits from the Abstraction class
+     *            ✓  b) has an method
+     *         Client class: 
+     *               a) uses a method in the Abstraction class
+     *            ✓  b) creates a Concrete Implementation instance
+     *               c) uses the field or property in Abstraction
+     */
```

Comment:
bij alle testcases controleren op br/ aan het einde van een regel

Reply:
Waarvoor? Deze staat niet in de docs, ik heb ze nu allemaal weg gehaald